andela/ah-frontend-poseidon
# #162162960: Setup SASS processing on the client.

**What does this PR do?**
Sets up SASS preprocessing on the application to enable the use of `.scss` and `.css` files on the client.

**Description of the tasks to be completed?**
- Add node-sass package to the application.
- Convert App.css file to App.scss file to test.

**How should this be manually tested?**
- Clone the repository
- Observe that we have `App.scss` and `index.css`
- git checkout ch-sass-setup-162162960 
- Run the app using `yarn start`
- The application should be running.

**What are the relevant pivotal tracker stories?**
#162162960
